src/ngraph/op/topk.hpp

```diff
@@ -119,12 +119,7 @@ namespace ngraph
             {
             public:
                 using SortType = TopKSortType;
-
-                enum class Mode
-                {
-                    MAX,
-                    MIN
-                };
+                using Mode = v1::TopKMode;
```

Comment:
Is this a backward compatible change?

Reply:
Since visibility and values of this enum are the same it's compatible in my opinion.